docs/api/modernization/overview.md

```diff
@@ -0,0 +1,10 @@
+## Modernization
+
+The Electron team is currently undergoing an initiative to modernize our API in a few concrete ways. These include: updating our modules to use idiomatic JS properties instead of separate `getPropertyX` and `setpropertyX`, converting callbacks to promises, and removing some other antipatterns present in our APIs. The current status of the Promise intiative can be tracked in the [promisification](promisification.md) tracking file.
```

Comment:
```suggestion
The Electron team is currently undergoing an initiative to modernize our API in a few concrete ways. These include: updating our modules to use idiomatic JS properties instead of separate `getPropertyX` and `setpropertyX`, converting callbacks to promises, and removing some other anti-patterns present in our APIs. The current status of the Promise intiative can be tracked in the [promisification](promisification.md) tracking file.
```